XDA_Scraper_Selenium.py

```diff
@@ -11,35 +11,36 @@
 url = "https://www.xda-developers.com/"
 
 total = []
-for page in range(1,pages):
+for page in range(1, pages):
 
     driver.get(url)
-    
+
     sections = driver.find_elements_by_class_name("layout_post_2")
 
     for section in sections:
 
         # print(section.text)
         articles = section.find_elements_by_class_name('item_content')
```

Comment:
looks good to me! the variables are well named so it is easy to read, but some comments wouldn't hurt as well.  to describe what the for looks are looping over. 
i.e. 
```
# loop over each section in the post layout and retrieve each item 
for section in sections:  
         .... 
````